beesdoo_shift/wizard/subscribe.py

```diff
@@ -26,6 +26,24 @@ def _get_date(self):
         else:
             return date
 
+    def _get_info_session_date(self):
+        date = self.env['res.partner'].browse(self._context.get('active_id'))\
+            .info_session_date
```

Comment:
Petite remarque de syntaxe. Selon les conventions de Python ([PEP8](https://legacy.python.org/dev/peps/pep-0008/)), on évite d'utiliser `\` pour couper des lignes, sauf quand on ne sait pas faire autrement.

On préfère utiliser les particularité de syntaxe pour réaliser des coupures de ligne.

Ici par exemple, j'écrirais :
```python
date = (self.env['res.partner']
        .browse(self._context.get('active_id'))
        .info_session_date)
```

Note l'utilisation des parenthèses. :)

Reply:
Right, you already told me, sorry :)
Done

Reply:
Moi j'aime bien aligner les `.`, je trouve que ça marche bien l'enchainement des opérations.

Reply:
done